docs/docs/reference/custom-components/05-aica-package-toml.md

```diff
@@ -262,6 +287,42 @@ python3 ${WORKSPACE}/src/my_component/script.py
 """
 ```
 
+### `[metadata]`
+
+Required.
+
+#### `[metadata.version]`
+
+Required. Must be a [semver-compliant](https://semver.org/) version string.
+
+```toml title="aica-package.toml"
+[metadata]
+version = "1.0.0"
+```
+
+#### `[metadata.description]`
+
+Optional. A short description of the component.
+
+```toml title="aica-package.toml"
+[metadata]
+description = "My awesome component"
+```
+
+#### `[metadata.collection]`
+
+Required when using multiple `[build.packages]`, otherwise ignored. This must contain the names of the collection being built (the set of multiple components).
+
+`[metadata.collection.name]` is required. It is the name of the collection.
+
+`[metadata.collection.ros-name]` is optional. This must be specified if `[metadata.collection.name]` is not a valid ROS package name.
+
+```toml title="aica-package.toml"
+[metadata.collection]
+name = "my-collection"
+ros-name = "my_collection" # required because `my-collection` is not a valid ROS package name
+```
+
 #### Advanced usage
 
 ##### `docker-image://`
```

Comment:
In this section and in build-context, you still refer to `[build.environment]` (`[build.environment.aica]` / `[build.environment.aica.libraries]` / `[build.environment.aica.ros]`)